src/website/views/components/popover/_examples.pug

```diff
@@ -286,6 +286,85 @@ h3 Open on hover
         });
       </script>
 
+h3 Close button
+.example.-mb--3
+  .-p--3
+    chi-button#button-example-popover-closable-1 Click Me!
+    chi-popover#popover-example-popover-closable-1(
+      title="Title"
+      variant="text"
+      arrow
+      reference="#button-example-popover-closable-1"
+      closable)
+      | Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+      | sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+  .example-tabs.-pl--2
+    ul.chi-tabs#examples-close-button
+      li.-active
+        a(href=`#webcomponent-close-button`) Web Component
+      li
+        a(href=`#html-close-button`) HTML Blueprint
+  .chi-tabs-panel.-active#webcomponent-close-button
+    .chi-tab__description
+      | Use the attribute <code>closable</code> to render the Popover with a close button.
+    :code(lang="html")
+      <chi-button id="button-example-popover-closable-1">With title</chi-button>
+      <chi-popover
+        id="popover-example-popover-closable-1"
```

Comment:
Thoughts on updating the formatting for consistency with other examples? We need to add tests too.